scripts/kf7281/main.py

```diff
@@ -0,0 +1,188 @@
+import logging
+
+from pathlib import Path
+from kfcicli.main import GitCredentials, KubeflowCI, Client
+from kfcicli.charms import LocalCharmRepo
+from kfcicli.utils import setup_logging
+import json
+import configparser
+import jinja2
+
+def reformat_tox(filename: Path):
+    """Import and export a tox file to get the formatting right.
+
+    This is a function with side-effects, that modify the underlying file.
+
+    Args:
+        filename: Path, name of the file to be re-formatted
+    """
+    config = configparser.ConfigParser()
+
+    config.read(filename)
+    with open(filename, 'w') as configfile:
+        config.write(configfile)
```

Comment:
I understand the need for it. Though,
1. (not a blocker): I 'm not a big fan of the formatting this introduces (everything at the same level) https://github.com/deusebio/katib-operators/pull/5/files#diff-c23978033532988c1a8d3bfd904cb4f31a7572458e6d252493da727663f3c8caR69-R70
2. I don't completely get what formatting rules this uses. Let's just ensure this aligns with the formatting the repo does ([example](https://github.com/canonical/admission-webhook-operator/blob/main/tox.ini#L45-L51)). EDIT: I noticed that [we don't format tox.ini](https://github.com/canonical/admission-webhook-operator/blob/6764bd5333a1e3d777f18f539111c15d039f0fc1/tox.ini#L13) files so we 'll be good

Reply:
I believe the new formatting is what is more natural to configparser, so I would personally use this. It is likely to be more "standard" than what we currently use